### PR TITLE
ELEMENTS-1439: add unit test to prevent regression of nuxeo-layout validation

### PR DESCRIPTION
**Pull request by richardsd**

### CODE REVIEW

File: ui/test/nuxeo-layout.test.js [hunk 1]
```diff
@@ -169,5 +180,81 @@ suite('nuxeo-layout', () => {
...
+
+      // the nuxeo-file should be invalid because it is still uploading
+      let validity = await layout.validate();
+      expect(validity).to.be.false;
+      expect(fileInput.invalid).to.be.true;
+
+      // when the file upload finishes, the nuxeo-file should be valid
+      fileInput.uploading = false;
+      validity = await layout.validate();
+      expect(fileInput.invalid).to.be.false;
```
Comment (semisse) [inline ui/test/nuxeo-layout.test.js:206]: should we also include this line as juxtaposition with the above?
```suggestion
      expect(validity).to.be.true;
      expect(fileInput.invalid).to.be.false;
```

**Review by semisse — DISMISSED** [review]
Thank you for this 🙏

File: ui/test/nuxeo-layout.test.js (hunk 1 above)
Comment (richardsd) [reply ui/test/nuxeo-layout.test.js:206]: Sure, it makes sense 👍 Thanks

**Review by Gabez0r — APPROVED** [review]

**Review by mnixo — APPROVED** [review]

**Comment by richardsd** [comment]
Cross repo build https://github.com/nuxeo/nuxeo-web-ui/runs/4518776124 ⌛

**Pull request merged**